server/jms/src/main/java/org/hawkular/apm/server/jms/CommunicationDetailsCacheMDB.java

```diff
@@ -67,7 +68,8 @@ public void init() {
         setProcessor(new AbstractProcessor<CommunicationDetails,Void>(ProcessorType.ManyToMany) {
 
             @Override
-            public List<Void> processManyToMany(String tenantId, List<CommunicationDetails> items) throws Exception {
+            public List<Void> processManyToMany(String tenantId, List<CommunicationDetails> items)
+                    throws RetryAttemptException {
                 communicationDetailsCache.store(tenantId, items);
```

Comment:
Is the lack of a try block here because the`StoreException` is never thrown by `CommunicationsDetailsCache` or have you forgotten to refactor this method?